k9mail-library/src/main/java/com/fsck/k9/mail/Address.java

```diff
@@ -32,20 +33,20 @@
 
     private String mPersonal;
 
-    public Address(Address address) {
+    public Address(@NonNull Address address) {
```

Comment:
I don't like adding `@NonNull` annotations because it should be the default. Rather add `@Nullable` when `null` is a valid option.

Reply:
I find `@NonNull`ing somewhat non obvious non-null's is more useful than getting caught up in ideals, but I'll remove it.